services/inc/services_dynalib.h

```diff
@@ -55,7 +56,7 @@ DYNALIB_FN(10, services, LED_Fade, void(Led_TypeDef))
 DYNALIB_FN(11, services, Get_LED_Brightness, uint8_t(void))
 
 DYNALIB_FN(12, services, set_logger_output, void(debug_output_fn, LoggerOutputLevel)) // Deprecated
-DYNALIB_FN(13, services, panic_, void(ePanicCode, void*, void(*)(uint32_t)))
```

Comment:
Does this break anything? only if someone is calling `panic_` directly, which they should not be doing?